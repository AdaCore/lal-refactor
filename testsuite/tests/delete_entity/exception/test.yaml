title: delete_exception

description: delete_enity tool test (exception declaration)

driver: shell_script
