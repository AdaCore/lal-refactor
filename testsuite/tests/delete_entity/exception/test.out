--  (main.orig) -L 2 -R 14
procedure Main is

begin
   begin
      null;
   exception
      when Constraint_Error =>
         raise;
   end;

   begin
      null;
   exception
      when Constraint_Error =>
         raise;
   end;

   begin
      null;
   exception
      when others =>
         raise;
   end;

   begin
      null;

   end;
end Main;
--  (main2.orig) -L 2 -R 14
procedure Main is
   E2 : exception;

begin
   begin
      null;
   exception
      when Constraint_Error =>
         raise;
   end;

   begin
      null;
   exception
      when Constraint_Error =>
         raise;
   end;

   begin
      null;
   exception
      when others =>
         raise;
   end;

   begin
      null;

   end;
end Main;
